### PR TITLE
Feature/shortcodes + tickettypes sync

### PR DESCRIPTION
**Pull request by menno-ll**
## Description

* Converts all vdm_ids to strings, because they can be strings
* Adds multiple shortcodes
* Add sync for tickettypes

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: app/API/Client.php [hunk 1]
```diff
@@ -40,10 +40,10 @@ public function get_events(): array {
 	/**
 	 * Get API event response object
 	 *
-	 * @param int $vdm_id
+	 * @param string $vdm_id
```
Comment (NielsdeBlaauw) [inline app/API/Client.php:43]: Wil je dit aub niet dubbel doen als er al een typehint in de method staat?

File: app/Objects/Event.php [hunk 2]
```diff
@@ -5,6 +5,27 @@
...
+			'post_status' => 'any',
+			'meta_query'  => array(
+				array(
+					'key'   => 'vdm_id',
+					'value' => $vdm_id,
+				),
+			),
+		);
+		$args         = wp_parse_args( $args, $default_args );
+		return static::get_one( $args );
```
Comment (NielsdeBlaauw) [inline app/Objects/Event.php:26]: Nice!

File: app/Objects/Event.php [hunk 3]
```diff
@@ -5,6 +5,27 @@
...
 	public static $type = 'll_vdm_event';
 
+	/**
+	 * Get object by vdm ID
+	 *
+	 * @param string $vdm_id
+	 * @param array $args
+	 * @return static|null
+	 */
+	public static function get_by_vdm_id( string $vdm_id, array $args = array() ) {
```
Comment (NielsdeBlaauw) [inline app/Objects/Event.php:15]:
```suggestion
	 */
	public static function get_by_vdm_id( string $vdm_id, array $args = array() ) :?Event {
```

**Review by NielsdeBlaauw — APPROVED** [review]
Lekker de methode om de `get_many` en `get_one` te gebruiken op deze manier. Super nice.

File: app/API/Client.php (hunk 1 above)
Comment (menno-ll) [reply app/API/Client.php]: https://github.com/level-level/ll-voordemensen/pull/11/commits/ace305a95f964f147acd639e323300f2fb975685

File: app/Objects/Event.php (hunk 3 above)
Comment (menno-ll) [reply app/Objects/Event.php]: https://github.com/level-level/ll-voordemensen/pull/11/commits/ace305a95f964f147acd639e323300f2fb975685

**Pull request merged**